crates/turborepo-lib/src/commands/login.rs

```diff
@@ -26,8 +26,18 @@ const DEFAULT_PORT: u16 = 9789;
 const DEFAULT_SSO_PROVIDER: &str = "SAML/OIDC Single Sign-On";
 
 pub async fn sso_login(base: &mut CommandBase, sso_team: &str) -> Result<()> {
+    let repo_config = base.repo_config()?;
     let redirect_url = format!("http://{DEFAULT_HOST_NAME}:{DEFAULT_PORT}");
-    let mut login_url = Url::parse(&format!("{}/api/auth/sso", base.repo_config()?.login_url()))?;
+    let login_url_configuration = repo_config.login_url();
+    let mut login_url = Url::parse(login_url_configuration)?;
+
+    login_url
+        .path_segments_mut()
+        .map_err(|_: ()| Error::LoginUrlCannotBeABase {
+            value: login_url_configuration.to_string(),
+        })?
+        .extend(["api", "auth", "sso"]);
+
     login_url
         .query_pairs_mut()
         .append_pair("teamId", sso_team)
```

Comment:
And yet we _also_ let you reuse your query parameters!

If your `LOGINURL` (née `loginUrl`) is `https://www.example.com/?myparam=1` you get this hilarious outcome:

`https://www.example.com/?myparam=1/api/auth/sso?myparam=1&teamId=vercel`.

Good luck figuring out how that happened as a user.

We should probably template this somehow instead, defining the data that we have. (Think: https://api.github.com/)

- `teamId`
- `redirect_url`